tests/legacy-cli/e2e_runner.ts

```diff
@@ -1,7 +1,5 @@
-// This may seem awkward but we're using Logger in our e2e. At this point the unit tests
```

Comment:
That comment is outdated.  The tests will work on a fresh git clone with yarn install.

Reply:
I think you're correct that this comment is wrong and this can remain as-is. Maybe initially I just had missing dependencies when converting e2e tests to bazel so I just switched to relative paths 🤷 😅 

Should I update the PR and just remove the comment? Or feel free to close it.

Reply:
It's good to merge; maybe update the message.  Technically probably better to not rely on unbuilt packages like that anyway.

Reply:
I think it only works do to the `lib/bootstrap-local.js` which compiles it at runtime and resolves the path using the `tsconfig-test.json`? Otherwise the comment would be true and it would depend on the package being built first?

Reply:
I've updated the commit message but I'm not sure if it is 100% correct?

Reply:
that works.